packages/react-components/stories/utils/query.ts

```diff
@@ -77,14 +77,14 @@ export const getTimeSeriesDataQuery = (
           {
             refId: '1',
             propertyId: propertyId1,
-            aggregationType: 'AVERAGE',
-            resolution: '1m',
+            // aggregationType: 'AVERAGE',
```

Comment:
will this break the story for anyone using the current implementation of useTimeSeriesData>

Reply:
No, this is just for storybook. I was testing raw vs aggregate data with the new implementation.